components/infobox/wikis/formula1/infobox_unit_car_custom.lua

```diff
@@ -77,6 +78,11 @@ function CustomUnit:setLpdbData(args)
 	local lpdbData = {
 		name = args.name,
 		type = 'car',
+		manufacturer = args.manufacturer
+		engine provider = args.engine
+		season(s) = args.season
+		team(s) = args.team
+		deasigner = args.designer
```

Comment:
has to go in extradata, and can't have parentheses unless you put them in as `['season(s)']`, but i'd suggest to not have parentheses in there.

Reply:
yeah () in keys is shit

also note that lpdb extradata keys should not contain `_` or spaces
just trim them away